src/lib.rs

```diff
@@ -67,8 +69,12 @@ pub fn parse_duration(string: &str) -> Result<i64, Error> {
     let mut neg = false;
 
     // Consume [-+]?
+
     if s != "" {
-        let c = s.chars().nth(0).unwrap();
+        let Some(c) = s.chars().nth(0) else {
```

Comment:
This seems unnecessary error handling, when s != "". The string is at least of length 1 and thus 0-th character exist.